docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,276 @@
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/vertiq_io_settings.png
+    :align: center
+    :scale: 50
+    :alt: Vertiq IO Submenu
+
+    Vertiq IO Submenu
+
+Now your Vertiq modules must be configured for proper communication with the flight controller.
+
+Configuring Your Vertiq Modules for Use with IFCI and PX4
+========================================================
+.. check which pulsing firmware we need.
+
+To use your Vertiq modules properly with IFCI 23-xx modules must be running speed firmware version 0.32 or newer, or pulsing firmware 0.27 or newer. Once flashed with appropriate firmware, connect each module **individually** to IQControlCenter and set the :blue:`UART Baud Rate` and the :red:`Module ID`. As stated previously, we recommend that you use a baud rate of 921600. Ensure that each module connected to the flight controller is set to a different, unique module ID.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/control_center_settings.png
+    :align: center
+    :scale: 50
+    :alt: Control Center Settings
+
+    Setting Baud Rate and Module ID
+
+With the modules set to unique module IDs, and the baud rate set to match the flight controller's, you can now connect your modules to the flight controller. To do this, find the flight controller's serial port configured to run the VERTIQ_IO module, and connect the TX of the serial port to each module's RX port. Connect the RX of the serial port to each module's TX port.
+
+.. figure:: ../_static/manual_images/pulsing_propeller/motor_serial_connection.png
+    :align: center
+    :alt: Bussed Serial Connection
+
+    Bussed Serial Connection
+
+Integration Setup
+=================
+
+.. warning::
+    Ensure that there are not propellers connected to the motor modules during this testing.
+
+With the modules connected to the flight controller and everything connected to a common ground power on the system, connect the flight controller to QGroundControl and go to the 'Parameters' section. Under the Vertiq IO submenu, set the ``Target_MODULE_ID`` parameter to the module ID of one of your modules and save. When you do this, the parameters should refresh and display the module's configured parameters. Any parameter that has a description starting with 'Module Param - ' is a value retrieved from the module.
+
+For each module ID that you have connected, set the following parameters to your desired values: ``CONTROL_MODE``, ``VERTIQ_FC_DIR``, ``VERTIQ_MOTOR_DIR``. Additionally for each module set the ``THROTTLE_CVI`` parameter to a different value. CVI stands for Control Value Index and more information can be found in :ref:`controlling_ifci`. Matching the module ID is a good way to do this, but if your module IDs do not start at 0, it is recommended that your CVIs do. If ``CONTROL_MODE`` is set to Voltage or Velocity, set the corresponding ``MAX_VOLTAGE`` or ``MAX_VELOCITY`` parameter as well.
```

Comment:
can you mention something about the trigger read guy in case people are having issues at all? it can be helpful in making sure everything is set correctly.